domains/Suraj.json

```diff
@@ -0,0 +1,31 @@
+{
+    "description": "portfolio",
+    "domain": "is-cool.dev",
+    "subdomain": "suraj",
+
+    "owner": {
+        "repo": "https://github.com/Clay990/Portfolio",
+        "email": "asuraj991m@gmail.com"
+    },
+
+"record": {
+        "A": ["1.1.1.1", "1.0.0.1"],
+        "AAAA": ["::1", "::2"],
+        "CNAME": "example.com",
+        "MX": ["mx1.example.com", "mx2.example.com"],
+        "TXT": ["example_verification=1234567890"],
+        "CAA": [
+            { "flags": 0, "tag": "issue", "value": "letsencrypt.org" },
+            { "flags": 0, "tag": "issuewild", "value": "sectigo.com" }
+        ],
+        "SRV": [
+            { "priority": 10, "weight": 60, "port": 5060, "target": "sipserver.example.com" },
+            { "priority": 20, "weight": 10, "port": 5061, "target": "sipbackup.example.com" }
+        ],
+        "PTR": [
+            "ptr.example.com"
+        ]
+    },
```

Comment:
These are all example records, please read: https://open-domains.net/docs/tutorial-basics/register

Reply:
You merged yesterday [Merge branch 'open-domains:main' into main](https://github.com/open-domains/register/pull/1560/commits/63285a48a990d682517863698807465c954f244d) this is not verifying, I put correctly I think plz check it 
![Screenshot (28)](https://github.com/user-attachments/assets/dfed6e39-9682-4245-be35-c560202e2e20)